src/Tools/ExternalAccess/Razor/IRazorDynamicFileInfoProvider.cs

```diff
@@ -17,16 +17,15 @@ internal interface IRazorDynamicFileInfoProvider
         /// <param name="projectId"><see cref="ProjectId"/> this file belongs to</param>
         /// <param name="projectFilePath">full path to project file (ex, csproj)</param>
         /// <param name="filePath">full path to non source file (ex, cshtml)</param>
-        /// <returns>null if this provider can't handle the given file</returns>
```

Comment:
So this comment was copied from the IDynamicFileInfoProvider interface which is the regular one -- however the wrapper never handled a null return from Razor, so this wasn't in practice the contract being enforced. I've updated this to match code behavior.